store/tikv/mocktikv/cop_handler_dag.go

```diff
@@ -95,7 +101,24 @@ func (h *rpcHandler) handleCopDAGRequest(req *coprocessor.Request) *coprocessor.
 		chunks = appendRow(chunks, data, rowCnt)
 		rowCnt++
 	}
-	return buildResp(chunks, err)
+	return chunks, errors.Trace(err)
+}
+
+func (h *rpcHandler) handleCopStream(req *coprocessor.Request) (tikvpb.Tikv_CoprocessorStreamClient, error) {
+	if len(req.Ranges) == 0 {
+		return nil, errors.Errorf("empty request range")
+	}
+	switch req.Tp {
+	case kv.ReqTypeDAG:
+		return h.handleCopStreamDAG(req)
+	case kv.ReqTypeAnalyze:
+	}
+	return nil, errors.New("not implement yet")
```

Comment:
s/not implement yet/unsupported cop request type/

Reply:
Theoretically, all kind of query push down are implemented in **coprocessor**,
but @lamxTyler make the analyze API a little different, I'm not sure the specific format, so leave it for him.
One other thing, analyze request doesn't use streaming yet.